packages/monaco/src/browser/monaco-diff-editor.ts

```diff
@@ -123,6 +128,22 @@ export class MonacoDiffEditor extends MonacoEditor {
     override shouldDisplayDirtyDiff(): boolean {
         return false;
     }
+
+    override handleVisibilityChanged(nowVisible: boolean): void {
+        if (nowVisible) {
+            this.diffEditor.setModel({original: this.originalTextModel, modified: this.modifiedTextModel});
+            this.diffEditor.restoreViewState(this.savedDiffState);
+            this.diffEditor.focus();
+        } else {
+            const originalModel = this.diffEditor.getOriginalEditor().getModel();
+            if (originalModel) {this.originalTextModel = originalModel; }
```

Comment:
Personally, I find the one-line `if`s very unreadable, I much prefer 
```typescript
if (originalModel) {
   this.originalTextModel = originalModel; 
}
```

but the coding guide does not say anything about that, so 🤷.

Reply:
I like one-liners, but I'm not that invested - three lines it is.